### ISSUE TITLE
drc-3.1.0 fails to install because of missing object 'bread'

### ISSUE THREAD

**Issue by rhurlin** [DoseResponse/drc#15]
Many thanks for the very nice package and the newest update to version 3.1.0. Unfortunately it fails to install with the following error message:

```
** testing if installed package can be loaded from temporary location
Error: package or namespace load failed for 'drc':
 object 'bread' not found whilst loading namespace 'drc'
Error: Laden fehlgeschlagen
Execution halted
ERROR: loading failed
* removing 'C:/Program Files/R/rw/library/drc'
* restoring previous 'C:/Program Files/R/rw/library/drc'
Error: Failed to install 'drc' from GitHub:
  (konvertiert von Warnung) installation of package ‘C:/Users/rhurlin/AppData/Local/Temp/RtmpsLoT5Y/file36a8508a31cc/drc_3.1-0.tar.gz’ had non-zero exit status
```

For some reason it does not found the newly introduced S3 method?

**Comment by OnofriAndreaPG** [comment]
Hello Rhurlin,
in my case a similar error was solved after updating R to 4.0.2. You might try this. Best Regards
Andrea

**Comment by rhurlin** [comment]
Hi Andrea,

Thanks for the reply. I have to admit, that I forgot to close this issue [SOLVED].

The error with missing object 'bread' was solved only several hours after reporting, see https://github.com/DoseResponse/drc/commit/131dc03c74e7000f583aa57c4e3325e889d68f11 and https://github.com/DoseResponse/drc/commit/be3955eddaf7d5f69d3ca1fbbac4fb4a5a435583.

Version 3.1-0 installs fine now via `install_githup("DoseResponse/drc")` on Windows, Linux and FreeBSD :)
